src/js/main.test.js

```diff
@@ -0,0 +1,53 @@
+var expect = chai.expect;
```

Comment:
Please confirm whether you need this line. I believe that when Karma is configured properly with the framework libraries, this line is handled for us elsewhere. If this line were to be mandatory in each test file, it would require many lines of duplication as the suites grow in size. 

For example, a frameworks property similar to below will create the `expect` for you.
```javascript
// karma.conf.js file
// ...
frameworks: ['mocha', 'chai', 'sinon-chai', 'dirty-chai'],
// ...
```